src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -62,6 +64,10 @@
      */
     private boolean wasTrendsFileSaved = false;
 
+    public void setPropertyFiles(List<String> propertyFiles) {
```

Comment:
it should be part of the configuration, not the report builder - sorry I did not notice that earlier

Reply:
also complete javadoc for new method with note about order

Reply:
next PR fixes this

Reply:
I expected this method will be removed

Reply:
yes it will be :)